tests/test_warm_reboot.py

```diff
@@ -789,3 +795,674 @@ def test_swss_port_state_syncup(dvs):
             assert oper_status == "down"
         else:
             assert oper_status == "up"
+
+
+#############################################################################
+#                                                                           #
+#                        Routing Warm-Reboot Testing                        #
+#                                                                           #
+#############################################################################
+
+
+def clear_logs(dvs):
+    dvs.runcmd(['sh', '-c', '> /var/log/syslog'])
+    dvs.runcmd(['sh', '-c', '> /var/log/swss/swss.rec'])
+    dvs.runcmd(['sh', '-c', '> /var/log/swss/sairedis.rec'])
+
+
+def start_zebra(dvs):
+    dvs.runcmd(['sh', '-c', 'supervisorctl start zebra'])
+
+    # Let's give zebra a chance to connect to FPM.
+    time.sleep(5)
+
+
+def stop_zebra(dvs):
+    dvs.runcmd(['sh', '-c', 'pkill -x zebra'])
+    time.sleep(1)
+
+
+def start_fpmsyncd(dvs):
+    dvs.runcmd(['sh', '-c', 'supervisorctl start fpmsyncd'])
+
+    # Let's give zebra a chance to connect to Zebra.
+    time.sleep(5)
+
+
+def stop_fpmsyncd(dvs):
+    dvs.runcmd(['sh', '-c', 'pkill -x fpmsyncd'])
+    time.sleep(1)
+
+
+def enable_warmrestart(dvs, db, app_name):
+    create_entry_tbl(
+        db,
+        swsscommon.CFG_WARM_RESTART_TABLE_NAME, app_name,
+        [
+            ("enable", "true"),
```

Comment:
Could you use CLI to config it as other test cases?
config warm_restart enable/disable [APP]

Reply:
Agree, that's the way to go.